lib/create-verify.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+
+function createVerify(referee) {
+    function verify(expected) {
+        if (
+            typeof expected !== "undefined" &&
+            (typeof expected !== "number" || expected < 1)
+        ) {
+            throw new TypeError("expected argument must be a number >= 1");
+        }
+
+        if (expected && referee.count !== expected) {
+            throw new Error(
+                "Expected assertion count to be " +
+                    expected +
+                    " but was " +
+                    referee.count
+            );
+        }
+
+        if (referee.count === 0) {
+            throw new Error(
+                "Expected assertion count to be at least 1, but was 0"
+            );
+        }
+
+        referee.resetCount();
```

Comment:
There is another edge case: If a test fails due `referee.count` not matching `expected`, then `count` is not reset to zero, potentially leaving an unwanted / unexpected side effect to a following test.

Reply:
Hmm ... I considered that it shouldn't "destroy the evidence", but of course we will be running tests after this.

Stateful libraries are messy.

I'll fix this